CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 All notable changes to the Lethe project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+### [Master] - 2025-10-16
+
+### Added
+
+- MAJOR A new application called lethe-particle-ray-tracing has been added. This application allows to trace rays through a domain containing particle and to extract the intersection point between those rays and particles. [#1629](https://github.com/chaos-polymtl/lethe/pull/1629)
```

Comment:
Can you add a line or two that explains what this can be used for ? Or like the intended use?

Since this will go in the release note, it is worth it to document it enough.